migrate/src/main/scala/migrate/utils/CoursierHelper.scala

```diff
@@ -0,0 +1,34 @@
+package migrate.utils
+
+import scala.concurrent.ExecutionContext
+
+import coursier.Repositories
+import migrate.CompatibleWithScala3
+import migrate.InitialLib
+import migrate.MigrateLib.Revision
+import scala.concurrent.ExecutionContextExecutor
+
+object CoursierHelper {
+  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
+
+  def get(lib: InitialLib): Seq[CompatibleWithScala3] = {
+    val libString = s"${lib.organization.value}:${lib.name.value}_3.0.0-M3:"
+    val res = coursier.complete
+      .Complete()
+      .withRepositories(Seq(Repositories.central))
```

Comment:
Should the repositories list be parameterized to reflect Repositories defined by the user?

Reply:
maybe..

Reply:
in other PR